This patch is part of a series of patches for speech-tools for
Debian GNU/Linux, which primarily fix build errors with GCC 4.3 and newer.

--- speech-tools-1.2.3.orig/include/EST_TSimpleVector.h
+++ speech-tools-1.2.3/include/EST_TSimpleVector.h
@@ -74,7 +74,7 @@
     void zero(void);
 
     /// Fill vector with default value
-    void empty(void) { if (*def_val == 0) zero(); else fill(*def_val); }
+    void empty(void) { if (*this->def_val == 0) zero(); else this->fill(*this->def_val); }
 };
 
 #endif
